src/text/3-applications-subsections/3D-analysis.tex

```diff
@@ -1,12 +1,26 @@
 \subsection{3D Analysis}
 \label{ssec:3d-analysis}
-
 \todo{Laura}
+The 1D analysis approach is a powerful tool to measure the spectrum of an isolated source. However, more complicated situations, such as field of views containing several overlapping sources, require a more careful treatment. Because it is based on integration over a region, the 1D approach can't disentangle the contribution of each source to the total flux in the selected region. Similarly, sources with extended or complex morphology can result in the measured flux being underestimated, and heavily dependent on the choice of extraction region.
+
+For such situations a more complex approach is needed, the so-called 3D analysis. The three relevant dimensions are the two spatial angular coordinates and an energy axis. In this framework, a combined spatial and spectral model (that is, a SkyModel, see Section~\ref{ssec:gammapy-modeling}) is fitted to the sky-maps that were previously derived from the data and bundled into a MapDataset (see Sections~\ref{ssec:gammapy-makers,ssec:gammapy-datasets}).
+
+Starting from the \irfs corresponding to the same three simulated \cta observations used in Section~\ref{ssec:1d-analysis}, we can create a MapDataset via the MapDatasetMaker. However, we will not use the simulated event lists provided by \cta but instead, use the method MapDataset.fake() to simulate measured counts from the combination of several SkyModel instances. In particular we simulate:
```

Comment:
Do all the 3 observations have the same IRF?

Reply:
Maybe mention that a `DL4` dataset is directly simulated

Reply:
about the IRFs: that's a good point, I think that is the case. But I am not 100% sure, also I don't think anything would change in the text if so. Or what do you propose?